Grafana Toolkit does not provide @typescript-eslint

I've upgraded toolkit to v9 and it fails with

> ✖ Failed to load plugin '@typescript-eslint' declared in '--config#overrides[0] » @grafana/eslint-config': Cannot find module '@typescript-eslint/eslint-plugin'

see https://github.com/simPod/GrafanaJsonDatasource/runs/6933292059?check_suite_focus=true.

To fix the issue on my side I had to install `@typescript-eslint/eslint-plugin`

--- a/package.json
+++ b/package.json
@@ -96,7 +96,7 @@
     "@emotion/eslint-plugin": "11.7.0",
     "@grafana/api-documenter": "7.11.2",
     "@grafana/e2e": "workspace:*",
-    "@grafana/eslint-config": "3.0.0",
+    "@grafana/eslint-config": "4.0.0",
     "@grafana/toolkit": "workspace:*",
     "@grafana/tsconfig": "^1.2.0-rc1",
     "@lingui/cli": "3.13.3",

--- a/packages/grafana-toolkit/package.json
+++ b/packages/grafana-toolkit/package.json
@@ -39,7 +39,7 @@
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "@grafana/data": "9.1.0-pre",
-    "@grafana/eslint-config": "^3.0.0",
+    "@grafana/eslint-config": "^4.0.0",
     "@grafana/tsconfig": "^1.2.0-rc1",
     "@grafana/ui": "9.1.0-pre",
     "@jest/core": "27.5.1",

